package.json

```diff
@@ -9,9 +9,9 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:ci": "jest --maxWorkers=2 --coverage",
-    "lint": "eslint --fix --ext .ts src/**",
-    "lint:check": "eslint --max-warnings 0 --ext .ts src/**",
-    "format": "prettier --write \"src/**/*\"",
+    "lint": "eslint --fix --ignore-path .lintignore --ext .ts src",
+    "lint:check": "eslint --ignore-path .lintignore --max-warnings 0 --ext .ts src",
+    "format": "prettier --write --ignore-path .lintignore \"src\"",
```

Comment:
`src/**` was causing these tools to try and find `.ts` files in the new src/__snapshots__ folder and failing.